test/Microsoft.DocAsCode.Metadata.ManagedReference.Tests/TripleSlashParserUnitTest.cs

```diff
@@ -100,17 +121,19 @@ static int Main()
             {
                 AddReferenceDelegate = null,
                 PreserveRawInlineComments = false,
+                Source = new SourceDetail()
+                {
+                    Path = Path.Combine(inputFolder, "Source.cs"),
```

Comment:
>Path [](start = 20, length = 4)

what docfx generates is path relative to docfx.json? this way seems it doesn't work ?